app/api/temp_messages/ping.rb

```diff
@@ -0,0 +1,8 @@
+module TempMessages
+  class Ping < Grape::API
+    desc 'Returns pong.'
+    get :ping do
+      { ping: params[:pong] || 'pong' }
```

Comment:
```suggestion
      { ping: 'pong' }
```

Reply:
jeje sorry it was playing with `/api/ping?pong=oli`

```json
{
    "ping": "oli"
}
```

removed!